jbx/recipes/default.rb

```diff
@@ -100,7 +100,7 @@
 # Run database migrations
 execute "database-migrations" do
   cwd "#{node["jbx"]["path"]}/application/cli"
-  command "php cli.php migrations:migrate --no-interaction"
+  command "./migration.sh -c migrate -d all -o --no-interaction"
```

Comment:
You should probably be explicit here just for more safety, but this is otherwise correct
`/bin/bash migration.sh`